core/templates/pages/contributor-dashboard-admin-page/contributor-admin-dashboard-page.component.ts

```diff
@@ -137,14 +145,19 @@ export class ContributorAdminDashboardPageComponent implements OnInit {
           }
         );
 
-        this.selectedLastActivity = 0;
-
         this.userService.getUserInfoAsync().then(userInfo => {
           const username = userInfo.getUsername();
 
           if (username === null) {
             return;
           }
+          this.today = new Date();
+          this.selectedFirstActivity = new Date();
+          this.selectedLastActivity = new Date(
+            this.today.getTime() - 24 * 60 * 60 * 1000 * 90
+          );
+          this.selectFirstActivity();
+          this.selectLastActivity();
```

Comment:
Remove these statements as per one of the comments above. `createFilter` is called later on within this method anyways.

Reply:
done